CHANGES.md

```diff
@@ -15,7 +15,9 @@ To be released.
  -  `Swarm<T>` class became to implement `IDisposable` again and should be
     disposed to clean up its internal resources. [[#485]]
  -  `IStore.IterateStateReferences()` method became to receive
-    `highestIndex`, `lowestIndex`, and `limit` parameters.  [[#447], [#545]]
+    `highestIndex`, `lowestIndex`, and `limit` parameters.  [[#447], [#545]
+ -  types of `IAction.PlainValue` and states were restricted to
+    `Bencodex.Types.IValue`.  [[#541], [#552]]
```

Comment:
It doesn't cover overall changes enough, like `IAction.LoadPlainValue()` method, `AccountStateGetter` delegate, and many more.